Cargo.toml

```diff
@@ -68,3 +68,12 @@ tokio-test = "0.4.2"
 [profile.release]
 debug = true
 split-debuginfo = "packed"
+
+# The following metadata is used by `cargo-binstall`, and should be synchronized
+# with `.github/workflows/release.yml`.
+[package.metadata.binstall]
+pkg-url = "{ repo }/releases/download/{ version }/{ name }-{ version }-{ target }.{ archive-format }"
+pkg-fmt = "tgz"
```

Comment:
@yodatak Have you tried running `binstall` without these lines? Based on my reading of the binstall docs it _seems_ like this configuration is all covered by binstall's defaults…